app/params/params.go

```diff
@@ -1,8 +1,48 @@
 package params
 
+import (
+	"github.com/cosmos/cosmos-sdk/types/address"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/classic-terra/core/v2/types/util"
+)
+
 // Simulation parameter constants
 const (
 	StakePerAccount           = "stake_per_account"
 	InitiallyBondedValidators = "initially_bonded_validators"
 	BondDenom                 = "uluna"
 )
+
+func init() {
+	SetAddressPrefixes()
+}
+
+// SetAddressPrefixes builds the Config with Bech32 addressPrefix and publKeyPrefix for accounts, validators, and consensus nodes and verifies that addreeses have correct format.
+func SetAddressPrefixes() {
```

Comment:
Remove also, please.

Reply:
Answered in this [one](https://github.com/classic-terra/core/pull/417#discussion_r1470985144).